approvaltests/docs/how_to/BddScenarios.md

```diff
@@ -0,0 +1,27 @@
+<a id="top"></a>
+
+# How to make a BDD Scenario using Approvals
+<!-- toc -->
+## Contents
+
+* [Introduction](#introduction)
+* [Sample Code](#sample-code)<!-- endToc -->
+
+## Introduction
+The idea is that you will end up with a descriptive approved file that reads like a BDD Scenario. You can show this to your less-technical collaborators, and they should be able to understand what scenario is being tested without needing to read the test sourcecode.
+
+It doesn't use Gherkin strictly, but you still get a scenario with a name, description, and Given, When, Then steps.
+
+## Sample Code
+
+I need some help with this part of the documentation. I'd like to include PrintableScenarioTest here as an example.
+<!-- snippet: bdd_test -->
```

Comment:
I think what you need is the following. These comments in the java file:
```java
// begin-snippet: MySnippetName
public class PrintableScenarioTest
{
  @Test
  public void printBDDScenario() throws Exception
  {
    PrintableScenario story = new PrintableScenario("Buy Groceries", "Bob puts two items in the shopping cart");
    User currentUser = new User();
    ShoppingCart shoppingCart = new ShoppingCart(currentUser);
    story.given(new Printable<>(currentUser, UserPrinter::print),
        new Printable<>(shoppingCart, ShoppingCartPrinter::print));
    story.when("Add oranges to the cart", () -> {
      shoppingCart.add("Oranges", 1, BigDecimal.valueOf(5));
      return null;
    });
    story.when("Add apples to the cart", () -> {
      shoppingCart.add("Apples", 3, BigDecimal.valueOf(4));
      return null;
    });
    Approvals.verify(story.then());
  }
}
// end-snippet
```

and then you can reference it here in markdown with `snippet: MySnippetName`